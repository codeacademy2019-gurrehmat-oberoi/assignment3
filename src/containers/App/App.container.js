import React, { Component } from 'react';
import Header from '../../components/Header/Header.component';
import AuthorGroup from '../../components/AuthorGroup/AuthorGroup.component';
import './App.css';

class App extends Component {
  render() {
    return (
      <div className="App">
        <Header />
        <AuthorGroup author="JK Rowling" />
        <AuthorGroup author="Sydney Sheldon" />
      </div>
    );
  }
}

export default App;
