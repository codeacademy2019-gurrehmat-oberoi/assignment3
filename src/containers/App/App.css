.App {
  background-color: #494748;
  color: white;
}