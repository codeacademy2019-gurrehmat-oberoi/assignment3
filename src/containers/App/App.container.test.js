import renderer from 'react-test-renderer';
import React from 'react';
import App from './App.container';

describe('App', () => {
  it('should render an App', () => {
    const tree = renderer.create(<App />).toJSON();
    expect(tree).toMatchSnapshot();
  });
});
