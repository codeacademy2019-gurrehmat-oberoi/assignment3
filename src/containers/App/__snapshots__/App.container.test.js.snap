// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`App should render an App 1`] = `
<div
  className="App"
>
  <header>
    <div
      className="hr"
    />
    <div
      className="headerContent"
    >
      <div
        className="logo"
      >
        BS
      </div>
      <div
        className="headerText"
      >
        The Book Shelf
      </div>
    </div>
    <div
      className="hr"
    />
  </header>
  <div
    className="authorGroup"
  >
    <div
      className="author"
    >
      JK Rowling
    </div>
    <div
      className="books"
    >
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
  </div>
  <div
    className="authorGroup"
  >
    <div
      className="author"
    >
      Sydney Sheldon
    </div>
    <div
      className="books"
    >
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
      <div
        className="book"
      >
        <img
          alt="Book cover"
          className="bookCover"
          src="cover_image.jpeg"
        />
        <div
          className="bookTitle"
        >
          Harry Potter and The Sorcerers Stone
        </div>
        <div
          className="bookAppreciation"
        >
          <div
            className="bookRating"
          >
            4.5
          </div>
          <div
            className="bookLikeButton"
          >
            <img
              alt="like book"
              src={
                Object {
                  "ReactComponent": [Function],
                  "__esModule": true,
                  "default": "heart-red.svg",
                }
              }
            />
          </div>
        </div>
      </div>
    </div>
  </div>
</div>
`;
